📝 Use in memory database for testing SQL in docs (#1223)
fastapi/fastapi: docs/en/docs/advanced/testing-database.md

First, we create a new database session with the new database.

> We'll use an in memory database that persists for the duration of the tests.

Reviewer: Maybe?
```suggestion
We'll use an in memory database that persists during the tests.
```
LGTM overall :+1:
